README.md

```diff
@@ -105,6 +105,10 @@ git checkout openshift-ansible-3.0.47-6
 ```
 
 ## Usage
+
+### Pick your AWS Region
+You have to pick which region to deploy your OpenShift installation.   Specify or modify 'run.py' with the correct AWS region and corresponding AMI. 
+
```

Comment:
You don't have to pick a region. You could use the default. Do **NOT** modify run.py.

Please refactor.